docs/composedb/guides/composedb-server/server-configurations.mdx

```diff
@@ -224,5 +225,29 @@ Note: Switching between data storage locations is an advanced feature and should
 
 The Ceramic State Store holds state for pinned streams and the acts as a cache for the Ceramic streams that your node creates or loads. To ensure that the data you create with your Ceramic node does not get lost you must pin streams you care about and you must ensure that the state store does not get deleted.
 
+## Metrics
+Metrics are a critical part of running a production Ceramic node. They allow you to monitor the health of your node and the network, and to debug issues when they arise.
+
+Js-ceramic produces metrics in the Prometheus format. You can configure your Ceramic node to expose these metrics on an HTTP endpoint, which can then be scraped by a Prometheus server.
+Alternatively, you can configure the Ceramic node to send metrics to an Opentelemetry collector endpoint.
+
+### Prometheus endpoint
```

Comment:
huh it would be nice if we could be consistent to provide both options in the environment - we would just need to add overrides here i think

https://github.com/ceramicnetwork/js-ceramic/blob/develop/packages/cli/src/ceramic-cli-utils.ts#L146

Reply:
Updated to use the config file with a reference to the envs in README